Tensile/Configs/rocblas_hpa_bfloat16_gemm_nn_inc1_asm_full.yaml

```diff
@@ -474,9 +474,441 @@ BenchmarkProblems:
+          - Exact: { sizes: [ 289, 128, 32, 768 ], stridesA: [ -1, 289, 221952 ], stridesB: [ -1, 768, 98304 ], stridesC: [ -1, 289, 36992 ], stridesD: [ -1, 289, 36992 ] } # gemm_strided_batched
+          - Exact: { sizes: [ 289, 160, 32, 768 ], stridesA: [ -1, 289, 221952 ], stridesB: [ -1, 768, 122880 ], stridesC: [ -1, 289, 46240 ], stridesD: [ -1, 289, 46240 ] } # gemm_strided_batched
+          - Exact: { sizes: [ 289, 192, 32, 768 ], stridesA: [ -1, 289, 221952 ], stridesB: [ -1, 768, 147456 ], stridesC: [ -1, 289, 55488 ], stridesD: [ -1, 289, 55488 ] } # gemm_strided_batched
+          - Exact: [ 1024, 10224, 1, 1024, 1024, 1024, 1024, 1024 ] #gemm
+          - Exact: [ 1024, 10192, 1, 1024, 1024, 1024, 1024, 1024 ] #gemm
+          - Exact: [ 1024, 10208, 1, 1024, 1024, 1024, 1024, 1024 ] #gemm
+          - Exact: [ 1024, 10224, 1, 4096, 1024, 1024, 1024, 4096 ] #gemm
+          - Exact: [ 1024, 10224, 1, 3072, 1024, 1024, 1024, 3072 ] #gemm
+          - Exact: [ 4096, 10224, 1, 1024, 4096, 4096, 4096, 1024 ] #gemm
+          - Exact: [ 1024, 10240, 1, 3072, 1024, 1024, 1024, 3072 ] #gemm
+          - Exact: [ 1024, 10192, 1, 3072, 1024, 1024, 1024, 3072 ] #gemm
+          - Exact: [ 4096, 10192, 1, 1024, 4096, 4096, 4096, 1024 ] #gemm
+          - Exact: [ 1024, 10192, 1, 4096, 1024, 1024, 1024, 4096 ] #gemm
+          - Exact: [ 1024, 10200, 1, 3072, 1024, 1024, 1024, 3072 ] #gemm
+          - Exact: [ 1024, 10184, 1, 1024, 1024, 1024, 1024, 1024 ] #gemm
+          - Exact: [ 4096, 10208, 1, 1024, 4096, 4096, 4096, 1024 ] #gemm
+          - Exact: [ 1024, 10208, 1, 3072, 1024, 1024, 1024, 3072 ] #gemm
+          - Exact: [ 1024, 10208, 1, 4096, 1024, 1024, 1024, 4096 ] #gemm
+          - Exact: [ 1024, 10224, 1, 2048, 1024, 1024, 1024, 2048 ] #gemm
+          - Exact: [ 1024, 10240, 1, 2048, 1024, 1024, 1024, 2048 ] #gemm
+          - Exact: [ 1024, 10120, 1, 1024, 1024, 1024, 1024, 1024 ] #gemm
+          - Exact: [ 1024, 10192, 1, 2048, 1024, 1024, 1024, 2048 ] #gemm
+          - Exact: { sizes: [ 64, 24, 6816, 24 ], stridesA: [ -1, 1308672, 192 ], stridesB: [ -1, 24, 576 ], stridesC: [ -1, 436224, 64 ], stridesD: [ -1, 436224, 64 ] } # gemm_strided_batched
+          - Exact: { sizes: [ 64, 24, 6816, 24 ], stridesA: [ -1, 1308672, 192 ], stridesB: [ -1, 24, 576 ], stridesC: [ -1, 1308672, 192 ], stridesD: [ -1, 1308672, 192 ] } # gemm_strided_batched
+          - Exact: [ 1024, 10152, 1, 1024, 1024, 1024, 1024, 1024 ] #gemm
+          - Exact: [ 1024, 10080, 1, 3072, 1024, 1024, 1024, 3072 ] #gemm
+          - Exact: { sizes: [ 64, 26, 6272, 26 ], stridesA: [ -1, 1204224, 192 ], stridesB: [ -1, 26, 676 ], stridesC: [ -1, 1204224, 192 ], stridesD: [ -1, 1204224, 192 ] } # gemm_strided_batched
+          - Exact: { sizes: [ 64, 26, 6272, 26 ], stridesA: [ -1, 1204224, 192 ], stridesB: [ -1, 26, 676 ], stridesC: [ -1, 401408, 64 ], stridesD: [ -1, 401408, 64 ] } # gemm_strided_batched
```

Comment:
1204224 sizes in strides for me requests ~350 GB ram for the entire benchmark. Not sure if this may be a bit big.

Reply:
1204224 which is 192x6272, it already takes batches into account